src/main/java/hw7/entities/Data.java

```diff
@@ -0,0 +1,67 @@
+package hw7.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Builder
+@Getter
+@AllArgsConstructor
+public class Data {
```

Comment:
What kind of "Data" is this ? You have to name the class with the reference to it purpose.